memory_task.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+#MEMORY TASK (BETA)
+from __future__ import division
+import os
+from PIL import Image
+import random
+from psychopy import core, visual, event #help(visual.ImageStim)
+
```

Comment:
Imports should be in 3 blocks:

System
3rd party (nibabel, dipy, etc)
Those related to the project (here, you have nothing else)